tabular/src/autogluon/tabular/models/fastainn/tabular_nn_fastai.py

```diff
@@ -458,6 +458,13 @@ def load(cls, path: str, reset_paths=True, verbose=True):
         from fastai.learner import load_learner
         model = super().load(path, reset_paths=reset_paths, verbose=verbose)
         if model._load_model:
+            # Need the following logic to allow cross os loading of fastai model
+            # https://github.com/fastai/fastai/issues/1482
+            import pathlib
+            import platform
+            plt = platform.system()
+            if plt != 'Windows':
+                pathlib.WindowsPath = pathlib.PosixPath
```

Comment:
Wouldn't we need to update this back to pathlib.WindowsPath after the load?

Reply:
Good call. Updated